adding some information one the legal setup on start page (#32)
ghga-de/docs.ghga.de: user_docs/index.md

Please reach out to the [GHGA Helpdesk](mailto:helpdesk@ghga.de) for any questions on GHGA.

!!! info "Funding"

>     The [GHGA Consortium](glossary/glossary.md#ghga-consortium) is a legally non-independent association of research institutions from all over Germany. It is funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) as part of  GHGA – The German Human Genome-Phenome Archive (www.ghga.de, Grant Number [441914366](https://gepris.dfg.de/gepris/projekt/441914366?context=projekt&task=showDetail&id=441914366&) (NFDI 1/1)).

Author: I gave it another try:
```suggestion
    The [GHGA Consortium](glossary/glossary.md#ghga-consortium) is not an independent legal entity but an association of research institutions from all over Germany. It is funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) as part of  GHGA – The German Human Genome-Phenome Archive (www.ghga.de, Grant Number [441914366](https://gepris.dfg.de/gepris/projekt/441914366?context=projekt&task=showDetail&id=441914366&) (NFDI 1/1)).
```

>     The [GHGA Consortium](glossary/glossary.md#ghga-consortium) is a legally non-independent association of research institutions from all over Germany. It is funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) as part of  GHGA – The German Human Genome-Phenome Archive (www.ghga.de, Grant Number [441914366](https://gepris.dfg.de/gepris/projekt/441914366?context=projekt&task=showDetail&id=441914366&) (NFDI 1/1)).

Author: I changed it even different and committed it again, see https://github.com/ghga-de/docs.ghga.de/pull/32/commits/204c3c39147a850e862cdfa3312989c02d6d568c.
Ok with this?

>     The [GHGA Consortium](glossary/glossary.md#ghga-consortium) is a legally non-independent association of research institutions from all over Germany. It is funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) as part of  GHGA – The German Human Genome-Phenome Archive (www.ghga.de, Grant Number [441914366](https://gepris.dfg.de/gepris/projekt/441914366?context=projekt&task=showDetail&id=441914366&) (NFDI 1/1)).

Reviewer: ok!
